src/Sarif.Driver/Sdk/EntryPointUtilities.cs

```diff
@@ -0,0 +1,111 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
```

Comment:
(I'd put this comment on the PR description but apparently that's not supported...) -- The description says you bumped the version, but I don't see that.